assets/src/dashboard/animations/animationParts/index.js

```diff
@@ -17,16 +17,18 @@
  * Internal dependencies
  */
 import { ANIMATION_TYPES } from '../constants';
-import { WAAPIBounce } from './bounce';
+import { AnimationBounce } from './bounce';
 
 function throughput() {
-  return ({ children }) => children;
+  return {
+    WAAPIAnimation: ({ children }) => children,
+  };
 }
 
-export function WAAPI(type, args) {
+export function AnimationPart(type, args) {
```

Comment:
I renamed `WAAPI` to `AnimationPart` since it can generically call up an animation which supports _both_ WAAPI and AMP animations.